services/QuillLMS/spec/controllers/api/v1/session_feedback_histories_controller_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+describe Api::V1::SessionFeedbackHistoriesController, type: :controller do
+  context "index" do
+    it "should return successfully - no history" do
+      get :index
+
+      parsed_response = JSON.parse(response.body)
+
+      assert_response :success
```

Comment:
I think we should discuss this as a group, but we're mixing minitests and specs together (this is probably on me for introducing minitest in the comprehension engine). We should probably stick to one syntax. (we can circle back and replace this, I don't want to hold up this PR).

Reply:
I just wrote a quick `sed` to fix this in my tests, which is where all of these were, so that should get us back to being consistent.